Hearthstone Deck Tracker/Controls/CardImage.xaml.cs

```diff
@@ -0,0 +1,90 @@
+﻿using System.ComponentModel;
+using System.Runtime.CompilerServices;
+using System.Windows;
+using System.Windows.Media.Animation;
+using Hearthstone_Deck_Tracker.Annotations;
+using Hearthstone_Deck_Tracker.Utility.Assets;
+
+namespace Hearthstone_Deck_Tracker.Controls
+{
+	/// <summary>
+	/// Interaction logic for CardImage.xaml
+	/// </summary>
+	public partial class CardImage : INotifyPropertyChanged
+	{
+		private string _cardId = "";
+		public string CardId
+		{
+			get => _cardId;
+			set
+			{
+				_cardId = value;
+				OnPropertyChanged();
+			}
+		}
+
+		private string _cardImagePath = null;
+		public string CardImagePath
+		{
+			get => _cardImagePath;
+			set
+			{
+				_cardImagePath = value;
+				OnPropertyChanged();
+			}
+		}
+
+		private string _createdByText = "";
+		public string CreatedByText
+		{
+			get => _createdByText;
+			set
+			{
+				_createdByText = value;
+				OnPropertyChanged();
+			}
+		}
+
+		private Visibility _createdByVisibility = Visibility.Visible;
+		public Visibility CreatedByVisibility
+		{
+			get => _createdByVisibility;
+			set
+			{
+				_createdByVisibility = value;
+				OnPropertyChanged();
+			}
+		}
+
+		public event PropertyChangedEventHandler PropertyChanged;
+
+		[NotifyPropertyChangedInvocator]
+		protected virtual void OnPropertyChanged([CallerMemberName] string propertyName = null)
+		{
+			PropertyChanged?.Invoke(this, new PropertyChangedEventArgs(propertyName));
+		}
```

Comment:
Since I remembered yesterday that it exists, let's just delete this and the `INotifyProperty` inheritance and replace it with `ViewModel`.